src/app/common/missinTableHeader.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Instead of disabling the rule, it would be better to [supply a text label](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/control-has-associated-label.md) to the empty `th` element.

Since there is no need of visible text there, you could supply an `aria-label` attribute to the element and set its value to a descriptive text, e.g. "Leave/Join Mission".

```
<th aria-label="Leave/Join Mission" />
```